packages/form-js-viewer/src/Form.js

```diff
@@ -17,7 +17,7 @@ import core from './core';
  * @typedef { import('didi').Injector } Injector
  *
  * @typedef { { [x: string]: any } } Data
- * @typedef { { [x: string]: string[] } } Errors
+ * @typedef { { [x: string]: any } } Errors

```

Comment:
What is the use-case for this one? What does _any_ represent?

Reply:
TypeScript started complaining after I changed the code of `submit`. This was the error message: 
![image](https://user-images.githubusercontent.com/7633572/122403505-b554e200-cf7e-11eb-9ff7-26938bd164db.png)

Reply:
Please revert this change and I'll figure it out.

Likely we simply need to change the signature of `set` or manually cast.

Reply:
Looking into this right now.

Reply:
The error doesn't appear after I changed the code to throw an error on submitting a read-only form.